src/client/network/client.cpp

```diff
@@ -53,16 +53,16 @@ void Client::sendChunkRequest(const ChunkPosition &position)
                             ENET_PACKET_FLAG_RELIABLE);
 }
 
-void Client::onPeerConnect(ENetPeer &peer)
+void Client::onPeerConnect(ENetPeer &peer) // FIXME: unreferenced formal parameter warning in MSVC
```

Comment:
For unused variables, this should be able to help with this

```suggestion
void Client::onPeerConnect([[maybe_unused]] ENetPeer &peer) 
```

https://en.cppreference.com/w/cpp/language/attributes/maybe_unused